packages/sitecore-jss/src/tracking/trackingApi.ts

```diff
@@ -8,52 +8,38 @@ import {
 } from './dataModels';
 import { TrackingRequestOptions } from './trackingRequestOptions';
 import querystring from 'querystring';
-import { HttpDataFetcher, HttpResponse } from './../data-fetcher';
-
-class ResponseError extends Error {
-  response: HttpResponse<unknown>;
-
-  constructor(message: string, response: HttpResponse<unknown>) {
-    super(message);
-
-    Object.setPrototypeOf(this, ResponseError.prototype);
-    this.response = response;
-  }
-}
-
-/**
- * @param {HttpResponse<T>} response response from fetch
- * @returns {HttpResponse<T>} response
- */
-export function checkStatus<T>(response: HttpResponse<T>) {
```

Comment:
also here, checkStatus should probably stay

Reply:
same reason as in the last comment.